Example/RSFormViewExampleApp/SignupExampleViewController.swift

```diff
@@ -23,32 +23,39 @@ class SignupExampleViewController: UIViewController {
     formView.delegate = self
     
     let configurator = FormConfigurator()
-    configurator.editingTitleColor = UIColor.astralBlue
-    configurator.editingBorderColor = UIColor.astralBlue
+    configureFormColors(formConfigurator: configurator)
+
+    formView.formConfigurator = configurator
+    configureButton()
+  }
+  
+  func configureFormColors(formConfigurator: FormConfigurator) {
```

Comment:
You can instantiate the formConfigurator and set it to the formView within this function, no need to pass it as a parameter